components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-profiles/reporting-profiles.component.html

```diff
@@ -22,9 +22,10 @@
     <chef-tbody *ngIf="!reportData.profilesListLoading">
       <chef-tr *ngFor="let profile of reportData.profilesList.items">
         <chef-td class="title-cell">
-          <chef-icon class="status-icon" [ngClass]="profile.status">
+          <chef-icon *ngIf="profile.status !== 'waived'" class="status-icon" [ngClass]="profile.status">
```

Comment:
maybe add a comment explaining that you need to do this b/c it's a custom icon?

Reply:
+1 to this.  I didn't realize what was going on until vj mentioned custom icon.

Reply:
Or can we have it added to the profile status icon list?  that would seem more maintainable

Reply:
@SEAjamieD the icon list is for material font icons, because it was material fonts I couldn't.